samples/mapzen-places-api-sample/src/main/res/layout/activity_autocomplete.xml

```diff
@@ -1,18 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:orientation="vertical"
     android:paddingBottom="@dimen/activity_vertical_margin"
     android:paddingLeft="@dimen/activity_horizontal_margin"
     android:paddingRight="@dimen/activity_horizontal_margin"
     android:paddingTop="@dimen/activity_vertical_margin"
     >
 
-  <fragment
-      android:id="@+id/place_autocomplete_fragment"
-      android:name="com.mapzen.places.api.ui.PlaceAutocompleteFragment"
```

Comment:
I think we should keep the `PlaceAutocompleteFragment` in this example since it shows off the reusable search bar UI component.

I'm guessing the reason you swapped the fragment for a custom button is to show using the `IntentBuilder` with an `AutocompleteFilter` but I think we can kill two birds with one stone here.

The `PlaceAutocompleteFragment` API should be updated to also allow setting an `AutocompleteFilter` (and `LatLngBounds`). Then internally the fragment can make use of `PlaceAutocomplete.IntentBuilder` to launch the `PlaceAutocompleteActivity` on tap.

This way we can test both implementations with a single demo activity.

What do you think?